### ISSUE TITLE
when fetching rows, show columns in the same order as in table, not alphabetical order

### ISSUE THREAD

**Issue by raghotham** [metabase/metabase#3167]
*  I am using the chrome browser.
*  My computer's OS is mac.
*  I'm running Metabase AWS [locally from the JAR / locally with the Mac App / on AWS / on Herkou / etc.] and it is using the postgres database.
*  My data is in a Redshift database. 
*  My Metabase version is 0.18.

**Comment by camsaul** [comment]
Hi @raghotham, it sounds like this is a duplicate of #2821. This was fixed by #2832, which shipped as part of Metabase 0.18.1. In #2931, which shipped as part of Metabase 0.19.0, we also added new options that let you order columns any way you like. [Upgrade to a newer version of Metabase](http://www.metabase.com/docs/latest/operations-guide/start.html#upgrading-metabase) and enjoy the new awesomeness! 

Closing as duplicate
